### ISSUE TITLE
NTR; pulmo: Presence of hyalinzed nodules

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6398]
**Preferred term label:**
Presence of hyalinzed nodules

**Synonyms**

**Definition (free text, please give PubMed ID)**
fibrosing nodules, consisting of either unilateral or bilateral central whorled deposits of lamellar collagen hyaline

PMID: 31538016

ISBN 978-1-4511-7643-8
Churg, Andrew, and Churg. Atlas of Interstitial Lung Disease Pathology : Pathology with High Resolution CT Correlations, Wolters Kluwer, 2013. 

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sarcoid Granulomas, Pulmonary hyalinizing granuloma (PHG) PMID: 31538016

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by mgriese** [comment]
Correct spelling:
Presence of hyalinized nodules

**Comment by pnrobinson** [comment]
@mgriese  We need to make the label comprehensible in isolation -- what about

Multiple pulmonary interstitial hyalinized nodules

**Comment by pnrobinson** [comment]
added
